maddymz/irondb
# Us1005

US adds validations to all editor flows.

To test:

1. git checkout US1005
2. Run ./iron.sh -lepmg to restart and populate all.
3. Navigate to localhost:3001/login in google chrome
4. Login using 'user1' and 'password'
5. Navigate to data-entry and select any of the editors.
6. Press the 'Validate' button.
7. the validations truth table should print below the form:

<img width="688" alt="Screen Shot 2019-04-21 at 1 33 59 PM" src="https://user-images.githubusercontent.com/26435230/56475208-45984a80-643a-11e9-8027-ad6680b14c7f.png">

8. Test validations script by re-entering values in the fields, re-submitting, and re-checking the truth table.
